snippets/csharp/System.Reflection.Metadata/Document/Program.cs

```diff
@@ -7,8 +7,12 @@ class Program
     {
         static void Main(string[] args)
         {
-            string pdbPath = Path.Combine(Path.GetDirectoryName(typeof(Program).Assembly.Location),"DocumentSnippets.pdb");
-            DocumentSnippets.ReadPdbDocuments(pdbPath);
+            string? loc = Path.GetDirectoryName(typeof(Program).Assembly.Location);
+            if (loc is not null)
```

Comment:
This could be rewritten as follows:

```suggestion
            if (Path.GetDirectoryName(typeof(Program).Assembly.Location) is string loc)
```